### PR TITLE
feat(config): add new parameters 17 and 18 for HeatIt TF016_TF021 FW 1.92

### PR DESCRIPTION
**Pull request by nesheimt**
Added support for new parameter(17 and 18) from v1.92

New datasheet attached

[HeatIt_1v9.pdf](https://github.com/user-attachments/files/17413128/HeatIt_1v9.pdf)

### CODE REVIEW

File: packages/config/config/devices/0x019b/tf016_tf021.json [hunk 1]
```diff
@@ -223,6 +223,26 @@
 			"minValue": 50,
 			"maxValue": 400,
 			"defaultValue": 210
+		},
+		{
+			"#": "17",
+			"$if": "firmwareVersion >= 1.92",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x019b/tf016_tf021.json:229]: Is this really firmware 1.92 and not 1.9? The manual you linked says "1.9"

**Review by AlCalzone — CHANGES_REQUESTED** [review]

File: packages/config/config/devices/0x019b/tf016_tf021.json (hunk 1 above)
Comment (nesheimt) [reply packages/config/config/devices/0x019b/tf016_tf021.json:229]: Hi. v1.92 is the version that reads out from the device(the manufacturer also refers to v1.92 when in contact with their service desk), snip from wave js us below:
image
But there are no v1.91 so 1.9 would also work in the "config code".

Comment (AlCalzone) [reply packages/config/config/devices/0x019b/tf016_tf021.json:229]: Ok then 1.92 is fine. Seems like another case of marketing versioning.

Comment (nesheimt) [reply packages/config/config/devices/0x019b/tf016_tf021.json:229]: Good! Should I press "Resolve conversation"(or something else) for this change to proceed?

Comment (AlCalzone) [reply packages/config/config/devices/0x019b/tf016_tf021.json:229]: No need. I'll finalize the PR once I'm back on the computer

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**